origin-dapp/src/pages/profile/Profile.js

```diff
@@ -163,6 +165,22 @@ class Profile extends Component {
         published: this.props.publishedProgress
       })
     }
+
```

Comment:
@sparrowDom can you throw in a comment here to explain what triggers these conditionals to be truthy? Is it just the window resizing?

Reply:
sure, yeah it is window resizing and opening/closing of the edit profile form